app/assets/javascripts/hqmf_util.js.coffee

```diff
@@ -712,7 +712,7 @@ class CrossProduct extends Array
 
 # Create a CrossProduct of the supplied event lists.
 XPRODUCT = (eventLists...) ->
-  hqmf.SpecificsManager.intersectAll(new CrossProduct(eventLists), eventLists)
+  hqmf.SpecificsManager.intersectAll(new CrossProduct(eventLists), eventLists, false, null, considerLeftMost: true)
```

Comment:
We don't want to considerLeftMost when calculating a boolean AND, chaining temporal operators, and probably other places. This should probably be refactored, but perhaps not now...